src/index.html

```diff
@@ -9,6 +9,13 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>Snake</h1>
+  <div class="container">
+    <div class="block block-1">1</div>
```

Comment:
```suggestion
    <div class="block">1</div>
```
Используй nth-child для стилизации, не надо добавлять отдельный класс